namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/elasticache.tf

```diff
@@ -0,0 +1,37 @@
+################################################################################
+# Interventions Elasticache for ReDiS
+################################################################################
+
+module "hmpps_interventions_elasticache_redis" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=4.2"
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  application            = var.application
+  environment-name       = var.environment-name
+  is-production          = var.is_production
+  infrastructure-support = var.infrastructure_support
+  team_name              = var.team_name
+  number_cache_clusters  = var.number_cache_clusters
+  node_type              = "cache.t2.small"
+  engine_version         = "4.0.10"
+  parameter_group_name   = "default.redis4.0"
```

Comment:
Do we need v4? The module defaults to v5

Reply:
the latest redis version is acutally 6.0 - might as well use that !